tests/unit/RegAddEditOrgPerson.spec.ts

```diff
@@ -363,44 +363,25 @@ describe('Registration Add/Edit Org/Person component', () => {
     wrapper.destroy()
   })
 
-  it('displays form data for proprietor-org (SP) - manual add', async () => {
-    const wrapper = createComponent(validProprietorOrg, -1, null)
-
-    await wrapper.find('.lookup-toggle').trigger('click')
-
-    expect(wrapper.find('.manual-add-article label').text())
-      .toContain('Edit Business or Corporation Not Registered in B.C.')
-    expect(wrapper.find('.manual-add-article p').text()).toContain('the Proprietor')
-
-    wrapper.destroy()
-  })
-
   it('displays form data for proprietor-org (SP) - business lookup', () => {
     const wrapper = createComponent(validProprietorOrg, -1, null)
 
-    expect(wrapper.find('.business-lookup-article label').text()).toContain('Business or Corporation Registered in B.C')
-    expect(wrapper.findAll('.business-lookup-article p').at(0).text()).toContain('the Proprietor')
+    expect(wrapper.find('.business-lookup-article label').text()).toContain('Business')
+    expect(wrapper.findAll('.business-lookup-article p').at(0).text()).toContain('Enter an existing B.C. business as the proprietor')
 
     wrapper.destroy()
   })
 
   it('displays form data for proprietor-org (SP) - edit', async () => {
     const wrapper = createComponent(validProprietorOrg, 0, null)
 
-    await wrapper.find('.lookup-toggle').trigger('click')
```

Comment:
remove `.lookup-toggle` because we do not use it anymore, and no confirm checkout, cannot let people to input their organzation name